lib/logos.spec.js

```diff
@@ -83,20 +83,20 @@ describe('Logo helpers', function () {
     })
 
     it('preserves color if medium logo on dark background', function () {
-      const logo = prepareNamedLogo({ name: 'android' })
+      const logo = prepareNamedLogo({ name: 'mastodon' })
       const decodedLogo = Buffer.from(
         logo.replace('data:image/svg+xml;base64,', ''),
         'base64',
       ).toString('ascii')
-      expect(decodedLogo).to.contain('fill="#34A853"')
+      expect(decodedLogo).to.contain('fill="#6364FF"')
     })
     it('preserves color if medium logo on light background', function () {
-      const logo = prepareNamedLogo({ name: 'android', style: 'social' })
+      const logo = prepareNamedLogo({ name: 'mastodon', style: 'social' })
```

Comment:
updated the test example here as https://github.com/simple-icons/simple-icons/pull/12834 changes the colour of the android icon such that it now _does_ get re-coloured on light